apps/govquests-frontend/src/domains/action_tracking/hooks/useCompleteActionExecution.ts

```diff
@@ -13,8 +13,9 @@ export const useCompleteActionExecution = (invalidateKey: string[]) => {
     CompleteActionExecutionVariables
   >({
     mutationFn: completeActionExecution,
-    onSuccess: (data, variables) => {
+    onSuccess: () => {
       queryClient.invalidateQueries({ queryKey: invalidateKey });
+      queryClient.invalidateQueries({ queryKey: ["notifications"] });
```

Comment:
we should probably invalidate both `invalidateKey` AND `"notifications"`, right?